shared/views/FilterCardView.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const { TemplateView, AvatarView } = require('hydrogen-view-sdk');
+const AvatarViewModel = require('../viewmodels/AvatarViewModel');
+
+class FilterCardView extends TemplateView {
```

Comment:
`FilterCardView` doesn't really describe what this single purpose component is for.

Name suggestion:

```suggestion
class NsfwCardView extends TemplateView {
```